packages/backend/src/auth/token/token.module.ts

```diff
@@ -0,0 +1,12 @@
+import {Module} from "@nestjs/common";
```

Comment:
jak to osobny modul to wyciagnij go z katalogu auth - ja bym go wcielil w modul auth